resources/ec2/v1alpha1/nat-gateway.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
this filename should be `resources/ec2/v1alpha1/natgateway.yaml` to match the resource name